src/Autocomplete/assets/src/controller.ts

```diff
@@ -149,6 +149,12 @@ export default class extends Controller {
             plugins.virtual_scroll = {};
         }
 
+        if ('plugins' in this.tomSelectOptionsValue && Array.isArray(this.tomSelectOptionsValue.plugins)) {
+            this.tomSelectOptionsValue.plugins.forEach((pluginName) => {
+                plugins[pluginName] = {};
+            });
+        }
```

Comment:
Given examples (https://tom-select.js.org/plugins/caret-position/ and https://tom-select.js.org/plugins/checkbox-options/) it looks like `plugins` can also be a key-value object.

Can't we use something like this?

```suggestion
        plugins = this.tomSelectOptionsValue.plugins || [];
```

Reply:
This one is treaky : https://tom-select.js.org/docs/plugins/ says we can put just plugin names (simple array) or plugin with options (object in js side). Also with simple assignment we override plugins configuration added line 138 to 150